handlers/misttriggers/recording_end.go

```diff
@@ -144,6 +144,15 @@ func (d *MistCallbackHandlersCollection) triggerRecordingEndSegmenting(w http.Re
 			return
 		}
 
+		// When createDtsh() completes issue another callback signaling to studio playback is ready
+		defer func() {
```

Comment:
Really nice use of `defer`! I like that it means we're not dependent on DTSH generation success to send the successful transcode callback